PluginsAndFeatures/azure-toolkit-for-intellij/src/com/microsoft/azure/hdinsight/serverexplore/ui/AddNewClusterFrom.java

```diff
@@ -157,19 +211,35 @@ protected void doOKAction() {
                         isCarryOnNextStep = false;
                     }
                 }
+
+                if (containersComboBox.getSelectedItem() == null) {
+                    errorMessage = "The storage container isn't selected";
+                    isCarryOnNextStep = false;
+                } else {
+                    storageContainer = ((BlobContainer) containersComboBox.getSelectedItem()).getName();
+                }
             }
 
             if (isCarryOnNextStep) {
                 getStorageAccount();
-            }
 
-            if (isCarryOnNextStep) {
-                if (storageAccount != null) {
+                if (storageAccount == null) {
+                    isCarryOnNextStep = false;
+                } else {
                     HDInsightAdditionalClusterDetail hdInsightAdditionalClusterDetail = new HDInsightAdditionalClusterDetail(clusterName, userName, password, storageAccount);
-                    ClusterManagerEx.getInstance().addHDInsightAdditionalCluster(hdInsightAdditionalClusterDetail);
-                    hdInsightModule.refreshWithoutAsync();
+                    try {
+                        JobUtils.authenticate(hdInsightAdditionalClusterDetail);
+
+                        ClusterManagerEx.getInstance().addHDInsightAdditionalCluster(hdInsightAdditionalClusterDetail);
+                        hdInsightModule.refreshWithoutAsync();
+                    } catch (Exception ignore) {
+                        isCarryOnNextStep = false;
+                        errorMessage = "Wrong username/password to log in";
```

Comment:
>errorMessage = "Wrong username/password to log in"; [](start = 24, length = 51)

Add authentication status code here for debugging? It might not be user name or password issue.

Reply:
Sure, fix it